builder/steps/gen-dockerfile/contents/test/detect_setup_test.ts

```diff
@@ -119,7 +119,7 @@ describe('detectSetup', () => {
               'version, see ' +
               'https://cloud.google.com/appengine/docs/flexible/nodejs/runtime'
         ],
-        {canInstallDeps: true, gotScriptsStart: false, useYarn: false});
+        {canInstallDeps: true, useYarn: false});
```

Comment:
nit: there test names include labels such as 'with start script'. This is not relevant anymore, so it would be good to remove from the test names.